src/pages/link-app/index.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  // AspectRatio,
+  Button,
+  Container,
+  createStyles,
+  Flex,
+  Grid,
+  Stack,
+  Text,
+  Title,
+  Image,
+} from '@mantine/core';
+// import { YoutubeEmbed } from '~/components/YoutubeEmbed/YoutubeEmbed';
+import { containerQuery } from '~/utils/mantine-css-helpers';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { NextLink } from '@mantine/next';
+import Lottie from 'react-lottie';
+import * as linkAnimation from '~/utils/lotties/link-animation.json';
+
+export default function LinkApp() {
+  const { classes } = useStyles();
+  const currentUser = useCurrentUser();
+  const isMember = currentUser?.isMember;
+  const buttonData = {
+    text: isMember ? 'Download the Link App' : 'Become a member',
+    href: isMember ? 'https://github.com/civitai/civitai-link-desktop/releases/latest' : '/pricing',
+  };
+
+  return (
+    <Container size="md">
```

Comment:
This feels small to me? Like it'll look bad on 4k devices 🤔

Reply:
Looks the same if I remove it so will do that. I think I pulled this from another page originally.

Reply:
I'm just saying it can be bigger really (lg/xl? maybe `fluid`?) Dunno how the assets are gonna look, but on a big screen, the MD has tons of space around

Reply:
I think its ok to have that space on the sides. Similar to leaderboard. The designs arent really meant to stretch out. Looks like removing it just defaults it to "md". I tried "xl" and its ok but kind of meh.